### PR TITLE
1.1.3b

### PR DESCRIPTION
**Pull request by anonymoussprocket**
### New Features

- Initial public release of "Login with Tezori/Galleon".
- Preview pending operations from the mempool.
- Cancel delegation by setting the delegate value to blank.

### Fixes

- Updated various dependencies, notably upgraded to electron 8.3.0 and reduced the number of linked (non-dev) dependencies from 219 to 151.
- Corrected behavior of "use max" function on the send view.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,8 +55,50 @@ To package for deployment for the local target:
...
-Change the `logo` field in `src/config.json`.
+### Contributing
+
+We welcome all contributions, be it issue reports, feature suggestions, [language files]() or pull requests. When submitting a PR expect the code to undergo a detailed review prior to any potential inclusion.
+
+## Support
+
+Cryptonomic hosts a [developer support channel]() on Riot. There is a Telegram chat for [Galleon user support](). Cryptonomic and several of its employees are active on Reddit and Twitter.
+
+[ConseilJS documentation]() and [Conseil documentation]() is quite through as well.
```
Comment (vishakh) [inline README.md:70]: The links are missing.

File: src/config.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "Tezori",
-    "version": "1.0.10b",
+    "version": "1.1.3b",
     "blockExplorerHost": "https://mininax.io",
```
Comment (vishakh) [inline src/config.json:4]: Is it time to switch to Arronax?

File: src/config.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
     "logo": "tezosLogo.png",
     "logoLink": "https://github.com/Cryptonomic/Tezori",
```
Comment (vishakh) [inline src/config.json:6]: Is this still the correct address?

File: src/locales/de.json [hunk 4]
```diff
@@ -20,7 +22,12 @@
             "proceed": "Fortfahren",
             "close": "Close",
             "withdraw": "Withdraw",
-            "deposit": "Deposit"
+            "deposit": "Deposit",
+            "mint": "Mint",
+            "burn": "Burn",
+            "sign": "Sign",
+            "verify": "Verify",
+            "copy": "Copy"
```
Comment (vishakh) [inline src/locales/de.json:30]: I am assuming these are placeholders. They don't look right to me.

**Comment by vishakh** [comment]
In general, I think we lack comments. At the minimum, each component and container should have at least a one-line comment stating what it is. It would also be good to have a diagram showing all the React components and how they are wired. It could simply be a screenshot from the React tab of Developer Tools from any browser.

File: src/config.json (hunk 2 above)
Comment (anonymoussprocket) [reply src/config.json:4]: no

**Pull request merged**